code/modules/surgery/organs/organ_internal.dm

```diff
@@ -15,10 +15,13 @@
 		insert(holder)
 	..()
 
+
 /obj/item/organ/internal/proc/insert(mob/living/carbon/M, special = 0)
 	if(!iscarbon(M) || owner == M)
 		return
 
+	processing_objects -= src
```

Comment:
As far as I know, organs already rot outside the body; after taking part in an experiment with another player ( @IcyV), multiple organs were removed from more than a few human mobs and they were most certainly rotting outside the body--stupidly quickly, at that, too.